### PR TITLE
Participation lists

### PR DESCRIPTION
**Pull request by belbel98**
have not explored front end for listing participation cards yet

### CODE REVIEW

File: src/main/java/seedu/address/model/participation/Participation.java [hunk 1]
```diff
@@ -32,6 +32,69 @@ public Participation(Person person, Competition competition, List<Attempt> attem
...
+        List<Attempt> topAttempts = new ArrayList<>(3);
+
+        // getting the top attempt for the squat
+        for (int i = 2; i >= 0; i--) {
+            Attempt squat = attempts.get(i);
+            assert squat.getLift() != Lift.SQUAT;
+            if (squat.getHasAttempted() && squat.getIsSuccessful()) {
+                topAttempts.add(squat);
+                break;
+            }
```
Comment (belbel98) [inline src/main/java/seedu/address/model/participation/Participation.java]: for some reason, this part of the code is never reached.. the conditions are never fulfilled..

File: src/main/java/seedu/address/model/participation/Participation.java [hunk 2]
```diff
@@ -32,6 +32,69 @@ public Participation(Person person, Competition competition, List<Attempt> attem
         this.attempts = attempts;
     }
 
+    /**
+     * Gets the top attempt of each of the 3 lifts.
+     * @return a list of three the top attempts for the squat, bench, deadlift respectively
+     */
+    public List<Attempt> getTopAttempts() {
+        List<Attempt> topAttempts = new ArrayList<>(3);
```
[the remark this answers is not in the document]
Comment (belbel98) [reply src/main/java/seedu/address/model/participation/Participation.java]: deleted this redundant method

File: src/main/java/seedu/address/model/participation/Participation.java [hunk 3]
```diff
@@ -32,6 +32,69 @@ public Participation(Person person, Competition competition, List<Attempt> attem
...
+                topAttempts.add(new Attempt(deadlift.getLift(), deadlift.getHasAttempted(),
+                        deadlift.getIsSuccessful(), 0));
+            }
+        }
+
+        return topAttempts;
+    }
+
+    public String getTopAttemptString() {
+        List<Attempt> topAttempts = this.getTopAttempts();
```
[the remark this answers is not in the document]
Comment (belbel98) [reply src/main/java/seedu/address/model/participation/Participation.java]: used getLiftScore for this one

File: src/main/java/seedu/address/model/participation/Participation.java [hunk 4]
```diff
@@ -32,6 +32,20 @@ public Participation(Person person, Competition competition, List<Attempt> attem
         this.attempts = attempts;
     }
 
+    /**
+     * Gets the highest score of each of the three lifts in this format: Squat/Bench/Deadlift.
+     * @return a string representation of the three lift score
+     */
+    public String getThreeLiftScore() {
```
Comment (ooimingsheng) [inline src/main/java/seedu/address/model/participation/Participation.java]: can you bring this lower and group it with all the other scorring methods in participation

**Pull request merged**